tiingo/api.py

```diff
@@ -82,6 +88,8 @@ def __init__(self, *args, **kwargs):
             'User-Agent': 'tiingo-python-client {}'.format(VERSION)
         }
 
+        self._frequency_pattern = re.compile('^[0-9]+(min)$|(hour)$', re.IGNORECASE)
```

Comment:
So close! I think you might have meant `[0-9]+(min|hour)$`, so that we can validate that the hour unit works.

We can test with: https://regex101.com

```
20min
1hour
```

Reply:
well it worked for the case I tested!  changed.